qualys-web-application-scanner/info.json

```diff
@@ -177,31 +184,31 @@
             "CANCELED"
           ],
           "name": "status",
-          "description": "The status of the scan: SUBMITTED, RUNNING, FINISHED, TIME_LIMIT_EXCEEDED, SCAN_NOT_LAUNCHED, SCANNER_NOT_AVAILABLE, ERROR or CANCELED"
+          "description": "Specify the status of the scan. You can choose from the following options: SUBMITTED, RUNNING, FINISHED, TIME_LIMIT_EXCEEDED, SCAN_NOT_LAUNCHED, SCANNER_NOT_AVAILABLE, ERROR or CANCELED."
         },
         {
           "title": "Authentication Status",
           "required": false,
           "editable": true,
           "visible": true,
-          "type": "select",
-          "option": [
+          "type": "multiselect",
+          "options": [
             "NONE",
             "NOT_USED",
```

Comment:
Change the option in the correct format like NOT_USED to Not Used in all

Reply:
This is a new version the options were added in the first release of connector. moreover, the customer is already used to the options.